.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
[OPTIONAL] To make your project professional, kindly add MIT license to your project.

Reply:
I will definitely do that @Rustamxon7, thanks.